test/functional/setup/setup_dbs.py

```diff
@@ -61,10 +63,13 @@ def setupAE(conn_options, dbname):
     if (args.AZURE.lower() == 'no'):
         manageTestDB('create_db.sql', conn_options, args.DBNAME)
 
+    print("About to set up databases...\n")
     # create tables in the new database
     setupTestDatabase(conn_options, args.DBNAME, args.AZURE)
+    print("About to populate tables...\n")
     # populate these tables
     populateTables(conn_options, args.DBNAME)
+    print("About to set up encryption...\n")
     # setup AE (certificate, column master key and column encryption key)
     setupAE(conn_options, args.DBNAME)
 
```

Comment:
This reminds me. This setup_dbs.py can be used to set up an Azure database as well. Do we need special handling with the AE part when it's Azure?